### ISSUE TITLE
NTR: Parvovirus IgM present in blood (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8079]
New term request Parvovirus IgM present in blood (hausma_130801143341): Antibodies

**Comment by pnrobinson** [comment]
@MickeySegal

Note we will be adding modifiers for serology.

Adding new term: 
- ID: HP:6000584
- label: Anti Parvovirus antibody positivity
- parents: [Positive bloodstream virus-specific antibody positivity (HP:0430059)](https://hpo.jax.org/app/browse/term/HP:0430059)
- pmid: 
- def: The presence of antibodies in the blood circulation that react against a component of parvovirus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8079
- synonyms:
